code/script.js

```diff
@@ -26,22 +28,110 @@ const showMessage = (message, sender) => {
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+  
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  step = 1;
+  showMessage(`Hello! Would you like to order coffee?`, 'bot')
+}
+
+const coffeePicker = (event) => {
+  event.preventDefault()
+  step ++
+  const value = document.getElementById('name-input').value;
+  showMessage(value, 'user')
+  if(value === 'yes') {
+    showMessage('Please select below', 'bot');   
+    inputWrapper.innerHTML = `
+      <div id="buttons" class="buttons">
+        <button value="coffee" id="coffeeBtn">Coffee</button>
+        <button value="latte" id="latteBtn">Latte</button>
+        <button value="cappuccino" id="cappBtn">Cappuccino</button>
+      </div>
+      `
+    document
+    .getElementById('coffeeBtn')
+    .addEventListener('click', () => { 
+      showMessage('Coffee', 'user')
+      showMessage('Ok, one coffee coming up for you', 'bot')
+      pickedCoffee = 'coffee'
+      inputWrapper.innerHTML = ''
+      setTimeout(() => chooseSize('coffee'), 1000)
+    })
+
+    document
+    .getElementById('latteBtn')
+    .addEventListener('click', () => {
+      showMessage('Latte', 'user')
+      showMessage('Ok, one latte coming up for you', 'bot')
+      pickedCoffee = 'latte'
+      inputWrapper.innerHTML = ''
+      setTimeout(() => chooseSize('latte'), 1000)
+    })
+
+    document
+    .getElementById('cappBtn')
+    .addEventListener('click', () => {
+      showMessage('Cappuccino', 'user') 
+      showMessage('Ok, one cappuccino coming up for you', 'bot')
+      pickedCoffee = 'cappuccino'
+      inputWrapper.innerHTML = ''
+      setTimeout(() => chooseSize('cappuccino'), 1000)
+      })
+  } else if(value === 'no'){
+      showMessage('Ok, see you another time!', 'bot');
+  } else {
+      showMessage("I'm sorry, I didn't understand your answer. Please answer 'yes' or 'no'.", 'bot');
+  }
+}
+ 
+const chooseSize = () => {
+  step ++
+  showMessage('What size do you want?', 'bot')
+  inputWrapper.innerHTML = `
+    <div id="sizeButtons" class="buttons">
+      <button value="short" id="shortBtn">☕ short</button>
+      <button value="tall" id="tallBtn">🥛 tall</button>
+    </div>
+    `
+  document
+  .getElementById('shortBtn')
+  .addEventListener('click', () => {
+    showMessage('Short', 'user')
+    pickedSize = 'short'
+    inputWrapper.innerHTML = ''
+    setTimeout(() => showPrice('short'), 1000)  
+  })
+
+  document
+  .getElementById('tallBtn')
+  .addEventListener('click', () => {
+    showMessage('Tall', 'user')
+    pickedSize = 'tall'
+    inputWrapper.innerHTML = ''
+    setTimeout(() => showPrice('tall'), 1000)  
+  })
+}
```

Comment:
It is a very nice and clean code 🦄 , but you repeat it a lot in all questions. Maybe you can find some similarities and try to build function similar to `showMessage` where you would receive text and options as parameters and just call it every time you need it? 🤓